### ISSUE TITLE
figcaption is being misused

### ISSUE THREAD

**Issue by GeorgeKerscher** [w3c/w3c-website#274]
_(Description updated to include example of URL where that issue occurs)_

On the beta website, a great deal of structured content, such as headings are in the figcaption element. This is a semantic misuse of the figcaption. Normally, the figcaption is used to provide additional information about an image, or an image group. This should be corrected.

Using NVDA, the major portion of the content is being presented as a caption. This means, that when an NVDA user presses the downarrow, they hear caption each time they press the key.

So, there are two issues, first the use of figcaption is semantically incorrect, and the screen reader (NVDA) users is annoyed by hearing caption before each line.

**URL**
The homepage: https://beta.w3.org/

**Comment by iadawn** [comment]
An example of this is:

<img width="1205" alt="image" src="https://user-images.githubusercontent.com/2444840/223708223-a121ac26-f56c-40a5-a255-65794a1c8b71.png">

In this example I have put a dashed border on the `<figure>` and a dotted border on the `<figcaption>` for illustration only.

Basically, the `<figure>` should be `<section>` and the `<figcaption>` just a `<div>`. There is some CSS associated with the `<figcaption>` but the CSS for the `<figure>` seems to be captured in classes.

**Comment by vivienlacourba** [comment]
Similar to #239

**Comment by NicolaSaunders** [comment]
@GeorgeKerscher Thank you for the insight into NVDA behaviour. I really appreciate it, and the annoyance of that extra announcement.

In light of the feedback here and in #239 I agree that I have overused the `figure` and `figcaption` elements for what could be called "teaser" content.

The markup for the "fifty-fifty" component at `/templates/components/fifty_fifty.html.twig` should be adjusted to replace the `<figure>` with `<section>` and replace the `<figcaption>` with `<div class="component--fifty-fifty__text">`.

Then, in `/assets-src/styles/sass/50-core-components/_fifty-fifty.scss` we can make the following changes to the Sass:

- Line 30: `.component--fifty-fifty figcaption {` becomes `.component--fifty-fifty__text {`
- Line 48: `.component--fifty-fifty figcaption .l-box {` becomes `.component--fifty-fifty__text .l-box {`
- Line 101: `figcaption` becomes `.component--fifty-fifty__text`
